stable_baselines3/common/save_util.py

```diff
@@ -308,28 +308,28 @@ def save_to_zip_file(
     :param pytorch_variables: Other PyTorch variables expected to contain name and value of the variable.
     :param verbose: Verbosity level: 0 for no output, 1 for info messages, 2 for debug messages
     """
-    save_path = open_path(save_path, "w", verbose=0, suffix="zip")
```

Comment:
The diff here is terrible, I only added a `with` to this line and indented the rest.

Reply:
Ack, thank you for the heads up.